.github/meta_issue_template/release-1.1.1-plugin-issues.md

```diff
@@ -0,0 +1,35 @@
+Coming from [opensearch-build#870](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+### You can find all the date in above issue 
+### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)
+
```

Comment:
Maybe include a 'How to use this issue' section, I think you could steal from https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md?plain=1#L11